swagger.go

```diff
@@ -19,22 +19,28 @@ type Swagger interface {
 	ReadDoc() string
 }
 
+// Documentation is an interface to register swagger document.
+type Documentation interface {
+	Register() (string, error)
+}
+
 // Register registers swagger for given name.
-func Register(name string, swagger Swagger) {
+func Register(name string, swagger Swagger) error {
```

Comment:
Changing the function signature introduces a breaking change.

Reply:
Isn't it better to allow user to handle error by himself instead of panicking?